tests/test_message.py

```diff
@@ -173,6 +173,11 @@ def test_text_html_simple(self):
         text_html = self.test_message.text_html
         assert text_html == test_html_string
 
+    def test_text_html_empty(self, message):
+        message.text = None
+        message.caption = "test"
+        assert not message.text_html
```

Comment:
Perhaps change to `assert message.text_html is None` ?

Reply:
same for the other tests of course

Reply:
I could, but why?

Reply:
Because it would properly test the feature. We don't want to allow empty strings, right?

Reply:
I don't see why not, actually I was think of doing:
```
if not message_text:
  return message_text
```